### PR TITLE
Type rule extraction

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31383

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+   "emailsubject": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailto": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
```
Comment (idovandijk) [inline Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json:401]: Why do we need to extract all indicator types from Email To?

File: Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json [hunk 2]
```diff
@@ -0,0 +1,710 @@
...
+   "sourcebrand": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "sourcehostname": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
```
Comment (idovandijk) [inline Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json:625]: Same as before - why extract all? Needs to have no extraction at all here

File: Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json [hunk 3]
```diff
@@ -0,0 +1,710 @@
...
+   "emailreceived": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": false,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailreplyto": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
```
Comment (idovandijk) [inline Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json:366]: Don't need to extract all - just email

File: Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json [hunk 4]
```diff
@@ -0,0 +1,710 @@
...
+   "emailbodyhtml": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
+    "extractIndicatorTypesIDs": [
+
+    ]
+   },
+   "emailcc": {
+    "extractAsIsIndicatorTypeId": "",
+    "isExtractingAllIndicatorTypes": true,
```
Comment (idovandijk) [inline Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json:303]: Don't need to extract all, just email

**Review by idovandijk — CHANGES_REQUESTED** [review]
Not sure if that's just how the file is, but when we looked together I didn't see that we extract all indicator types from things like "email to". I don't see any specific extractions that you did when we looked together. Bug?

File: Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json (hunk 1 above)
Comment (David-BMS) [reply Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json:401]: email, domain, account no?

Comment (idovandijk) [reply Packs/Phishing/IncidentTypes/incidenttype-Phishing_6.1.json:401]: Yes you're right actually. But isn't email = account? `Account.Email.Address`

File: Packs/Malware/pack_metadata.json [hunk 5]
```diff
@@ -1,8 +1,8 @@
 {
     "name": "Malware",
-    "description": "This popular “whack-a-malware” Content Pack helps you automate IOC extraction and enrichment, detonate malicious files, hunt for more IOCs and more.",
+    "description": "This popular \u201cwhack-a-malware\u201d Content Pack helps you automate IOC extraction and enrichment, detonate malicious files, hunt for more IOCs and more.",
     "support": "xsoar",
-    "currentVersion": "1.2.4",
+    "currentVersion": "1.2.5",
```
Comment (yaakovi) [inline Packs/Malware/pack_metadata.json:5]:
```suggestion
    "currentVersion": "1.2.5",
    "serverMinVersion": "6.0.0",
```

File: Packs/Phishing/pack_metadata.json [hunk 6]
```diff
@@ -2,7 +2,7 @@
     "name": "Phishing",
     "description": "Phishing emails still hooking your end users? This Content Pack can drastically reduce the time your security team spends on phishing alerts.",
     "support": "xsoar",
-    "currentVersion": "2.0.0",
+    "currentVersion": "2.0.1",
```
Comment (yaakovi) [inline Packs/Phishing/pack_metadata.json:5]:
```suggestion
    "currentVersion": "2.0.1",
    "serverMinVersion": "6.0.0",
```

**Review by idovandijk — APPROVED** [review]

**Pull request merged**